frontend/vite.config.ts

```diff
@@ -12,6 +12,11 @@ export default defineConfig({
   build: {
     sourcemap: true, // Source map generation must be turned on
   },
+  esbuild: {
+    supported: {
+      "top-level-await": true,
```

Comment:
Yes, exactly. I guess we could also move the session store initialisation, but it seems to me that this should not be an issue based on https://caniuse.com/mdn-javascript_operators_await_top_level stats. I only found this problem because the pipeline failed [here](https://caniuse.com/mdn-javascript_operators_await_top_level).